tests/test_parameters.py

```diff
@@ -1864,6 +1864,28 @@ def expected_func(timestep, scenario_index):
 
         m.run()
 
+    @pytest.mark.parametrize(['year'], [("2015",), ("2016",)])
+    def test_residual_days(self, year):
+
```

Comment:
This might benefit from a quick explanation / docstring

Reply:
done